lib/dor/datastreams/rights_metadata_ds.rb

```diff
@@ -121,6 +126,15 @@ def self.upd_rights_xml_for_rights_type(rights_xml, rights_type)
           loc_node = Nokogiri::XML::Node.new('location', rights_xml)
           loc_node.content = rights_type.split(':').last
           machine_node.add_child(loc_node)
+        elsif rights_type.start_with?('cdl')
+          cdl_node = Nokogiri::XML::Node.new('cdl', rights_xml)
+          if rights_type.end_with?('-nd')
```

Comment:
Are there any cases where this is not always true?

Reply:
if we only have cdl-stanford-nd, that is true that we don't this, and can add it back later if we need to support that distinction